ros/src/visualization/rviz_site_churchlot.rviz

```diff
@@ -21,13 +21,14 @@ Panels:
   - Class: rviz/Views
     Expanded:
       - /Current View1
+      - /SavedOrbit1/Focal Point1
```

Comment:
I'm not seeing another reference to this saved orbit in the commit, but I'll assume that isn't going to cause any issues with visualizations.

Reply:
Good point, thanks.  RViz is pretty finicky in how it saves its config and I think this is just an artifact left over from when I was deleting and remaking a saved view.  I added a commit to resave the config file after cleaning up the delta.